### PR TITLE
Contribution Templates

### PR DESCRIPTION
**Pull request by Kalman98**
GitHub really wanted me to make issue and pull request templates, and it seemed like a reasonable suggestion, so here they are.

### CODE REVIEW

File: PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -0,0 +1,9 @@
+Fixes # .
+
+
+Changes proposed in this pull request:
+-
+-
+-
+
+@teammodding/full-members
```
Comment (ParkerMc) [inline PULL_REQUEST_TEMPLATE.md]: What's this line for?

**Review by ParkerMc — COMMENTED** [review]
One thing.

File: PULL_REQUEST_TEMPLATE.md (hunk 1 above)
Comment (Kalman98) [reply PULL_REQUEST_TEMPLATE.md]: I'm assuming you mean the bottom one. It should ping all full members of the TEAMModding organization so they know to look at the pull request.

Comment (ParkerMc) [reply PULL_REQUEST_TEMPLATE.md]: People already get notified if they set up the notifications. Do we need it to also ping everyone?

Comment (Kalman98) [reply PULL_REQUEST_TEMPLATE.md]: No, but the GitHub template suggested it. I'll remove it.

**Review by ParkerMc — APPROVED** [review]

**Comment by Kalman98** [comment]
Thanks!

**Pull request merged**